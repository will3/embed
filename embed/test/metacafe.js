const test = require('./utils/test');

test('metacafe',
  'http://www.metacafe.com/watch/11506159/crazy-chick-let-me-see-your-phone-or-i-ll-jump');
