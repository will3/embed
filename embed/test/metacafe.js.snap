// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` metacafe 1`] = `
Object {
  "description": "She threatens to jump off this bridge if her boyfriend won't let her go through his phone.",
  "originalUrl": "http://www.metacafe.com/watch/11506159/crazy-chick-let-me-see-your-phone-or-i-ll-jump",
  "site_name": "Metacafe",
  "thumbnail": Object {
    "url": "http://cdn.metacafe.com/contents/videos_screenshots/11506000/11506159/preview.jpg",
  },
  "title": "
							Crazy Chick Let Me See Your Phone or I ll Jump
			",
  "video": Object {
    "description": undefined,
    "height": "315",
    "title": undefined,
    "url": "http://www.metacafe.com/embed/11506159/crazy-chick-let-me-see-your-phone-or-i-ll-jump/",
    "width": "560",
  },
}
`;
